src/Kdyby/Doctrine/Diagnostics/Panel.php

```diff
@@ -314,10 +314,7 @@ public function renderQueryException($e)
 			$types = $params = [];
 
 			if ($this->connection !== NULL) {
-				if (!$e instanceof Kdyby\Doctrine\DBALException || $e->connection !== $this->connection) {
-					return NULL;
```

Comment:
Are you sure that this can be removed?

Reply:
Yes, I'm.
There is condition `$e instanceof PDOException`. It means, $e cannot be instance of Kdyby\Doctrine\DBALException.